lib/Dialect/Comb/CombOps.cpp

```diff
@@ -21,6 +21,9 @@ using namespace comb;
 /// Return true if the specified type is a signless non-zero width integer type,
 /// the only type which the comb ops operate.
 static bool isCombIntegerType(mlir::Type type) {
+  if (auto typeAlias = type.dyn_cast<hw::TypeAliasType>())
+    type = typeAlias.getCanonicalType();
```

Comment:
nit: I'm not a huge fan of re-assigning variables like this, esp when recursion is clearly possible. (Doubly so when it's tail recursive.) I do it, but only when it's not simple to avoid it.